Library/views.py

```diff
@@ -118,10 +127,21 @@ def admin_dashboard():
         return render_template(
             "admin_dashboard.html", books=books, year=datetime.now().year
         )
-    flash("No books are there in library!")
+    flash("No books are there in library! Add Some")
     return render_template("admin_dashboard.html", year=datetime.now().year)
 
 
+class ShowStudentView(MethodView):
+    def get(self):
+        students = User.query.filter_by(admin = False).all()
+        print(students)

```

Comment:
When you are finished implementing remove all the print statements

Reply:
OK